brainbeats-backend/Controllers/UserController.cs

```diff
@@ -8,13 +11,33 @@ namespace brainbeats_backend.Controllers {
   [Route("api/[controller]")]
   [ApiController]
   public class UserController : ControllerBase {
+    [HttpPost]
+    [Route("login_user")]
+    public async Task<IActionResult> LoginUser(dynamic req) {
+      JObject body = DeserializeRequest(req);
+
+      try {
+        string res = await AuthConnection.Instance.LoginUser(body.GetValue("email").ToString(), 
+          body.GetValue("password").ToString());

```

Comment:
So you're storing the hashed password that the client side hashes right?

Have @hungnguyen1895 and @lloydoad communicated on using the same hash function?

Reply:
That's correct